src/theme/index.ts

```diff
@@ -3,7 +3,11 @@ import { createBreakpoints } from "@chakra-ui/theme-tools"
 import { Button } from "./Button"
 import { Card } from "./Card"
 
-// 2. Update the breakpoints as key-value pairs
+const config = {
+  initialColorMode: "dark",
+  useSystemColorMode: true,
```

Comment:
Should probably be `false` to not rely on the browser/system settings and force the `dark` mode.